governance/handler.go

```diff
@@ -127,40 +127,80 @@ func (g *Governance) AddVote(key string, val interface{}) bool {
 
 func (g *Governance) adjustValueType(key string, val interface{}) interface{} {
 	k := GovernanceKeyMap[key]
-	reqType := GovernanceItems[k].t
 	var x interface{}
 
 	// When an int value comes from JS console, it comes as a float64
-	if reqType == uint64T && reflect.TypeOf(val) == float64T {
-		x = uint64(val.(float64))
-		if float64(x.(uint64)) == val.(float64) {
+	if GovernanceItems[k].t == uint64T {
+		v, ok := val.(float64)
+		if !ok {
+			return val
+		}
+		x = uint64(v)
+		if float64(x.(uint64)) == v {
 			return x
 		}
 		return val
 	}
```

Comment:
```suggestion
		if float64(v.(uint64)) == v {
			return v
		}
		return val
	}
```

Reply:
The suggested code is not working so, I refactored the code like below.
```
		if float64(uint64(v)) == v {
			return uint64(v)
		}
```

Also, after refactoring like this, `x` is no longer needed, so `x` is removed.

Reflected at https://github.com/klaytn/klaytn/pull/1110/commits/fc1a4dd9144c6b78bef9ff326d804c47d7ea0b95. Please take another look.